src/apps/users/forms.py

```diff
@@ -113,10 +113,10 @@ class Meta(UserCreationForm.Meta):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         privacy_policy_url = self.get_privacy_policy_url()
-        privacy_policy_link = '<a href="{}" class="text-primary-500 font-bold hover:underline" target="_blank">política de privacitat.</a>'.format(  # noqa: E501
+        privacy_policy_link = '<a href="{}" class="text-primary-500 font-bold hover:underline" target="_blank">privacy policy</a>'.format(  # noqa: E501
             privacy_policy_url
         )
```

Comment:
```suggestion
        privacy_policy_link = (
            '<a href="{}" class="text-primary-500 font-bold hover:underline" '
            'target="_blank">{}</a>').format(
            privacy_policy_url,
            _("privacy policy"),
        )
```
No veig motiu per deixar la línia llarga i posar NOQA. La tallem en trossos petits.
El que ajuda és fer això de posar-la en parèntesis, així guanyes més espai per la string ja que la string comença a la línia de sota.

Afegeixo també que "privacy policy" sigui una string traduïda.